src/patternfly/components/Label/label-content.hbs

```diff
@@ -1,13 +1,23 @@
 {{#if label--IsEditable}}
   {{#> label-editable-content}}{{> @partial-block}}{{/label-editable-content}}
 {{else}}
-<{{#if label-content--IsLink}}a{{else}}span{{/if}} class="{{pfv}}label__content{{#if label-content--modifier}} {{label-content--modifier}}{{/if}}"
+<{{#if label-content--IsLink}}a{{else if label-content--IsAction}}button{{else}}span{{/if}} class="{{pfv}}label__content{{#if label-content--modifier}} {{label-content--modifier}}{{/if}}"
   {{#if label-content--IsLink}}
     href="#"
+    {{#if label--IsDisabled}}
+      tabindex="-1"
+      aria-hidden="true"
```

Comment:
Might be better to use aria-disabled here instead. Just thinking if it would be confusing for there to be a "close" button for something that is hidden from AT.

Reply:
🤦‍♂️ totally meant for that to be `aria-disabled`, nice catch.